### ISSUE TITLE
fish: brace expansion: character sequence

### ISSUE THREAD

**Issue by GReagle** [clarkgrubb/hyperpolyglot#43]
In Brace, Tilde, Command, and Pathname Expansion, the table is correct that fish does not have a built-in way to create a sequence from 1 to 10, but I think it would be more useful and informative to replace "none" with `seq 1 10`.

**Comment by limeblack** [comment]
@GReagle If that is not a built in is that something like busybox then?

**Comment by GReagle** [comment]
@limeblack Your questions doesn't make sense to me.  Could you elaborate or clarify please?

**Comment by limeblack** [comment]
@GReagle [busybox](http://www.busybox.net/) is a set of commands for typically bash.

If `seq 1 10` isn't a built in command where is the program/function being stored.

**Comment by GReagle** [comment]
Yes I know what busybox is.  It includes many commands.  It happens to include seq, but it also happends to include ls.  It is not really relevant though.  I am not referring to busybox's seq, I am just referring to the command seq, which on my computer is at /usr/bin/seq.  It is just an external command.  Maybe you have never heard of the seq command except in the context of busybox, but it is a standard Unix command like ls and env.
